zipline/zipline_deprecation_warning.py

```diff
@@ -13,20 +13,6 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from cpython cimport bool
```

Comment:
Looks like `git` thought this was a rename, when it is not.

Why remove the `pxd`?